app/user.go

```diff
@@ -146,7 +146,7 @@ func (c *UserClient) inviteUsers(
 	var roleIDs []string
```

Comment:
just so i understand--we are allowing a state where we can set account role to nil--and namespace roles to not nil?

Reply:
Also do we need to update the access package validation to allow the account role to be empty?

Reply:
Yes, account role can be nil, and namespace roles can be empty or not empty. 

The access validation already supports no account role if the user is a SCIM user.